pkg/server/tokenrequest/tokenrequest.go

```diff
@@ -224,6 +225,26 @@ const cssStyle = `
 </style>
 `
 
+const jsHandleCopyToClipboard = `
+<script>
+  const isCopyButton = (node) => node.localName === "button"
+    && node.attributes.getNamedItem("role") !== null
+    && node.attributes.getNamedItem("role").value === "copy-to-clipboard";
+
+  document.querySelectorAll("button[role=copy-to-clipboard]").forEach((button) => {
+    button.onclick = () => {
+      const textToCopy = Array.from(button.parentElement.childNodes.values()).reduce((acc, node) => {
+        return isCopyButton(node) ? acc : acc + node.textContent;
+      }, "");
```

Comment:
This works, but I wonder if it's not simpler to do something like

```
navigator.clipboard.writeText('oc login --token={{.AccessToken}} --server={{.PublicMasterURL}}');
```

in the button click handler. It repeats the command, but seems less error-prone.

Reply:
Fair point, I'll update the code and try to reduce the accidental complexity in JS code.

Reply:
As for the reasoning behind the code - I wanted to keep it simple - the given `<pre>` element contains the snippet we want to be copied, so we add `<button>` right into it and the button takes the content of the `<pre>` element (minus the "Copy" button text) and copies it to clipboard.